graphql-ballerina/src/graphql/parser/constants.bal

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// Special Characters
+const QUOTE = "\"";
+const BACK_SLASH = "\\";
+const DECIMAL = ".";
+
+const HASH = "#";
+
+const COMMA = ",";
+
+// Punctuators
+const EXCLAMATION = "!";
+const DOLLAR = "$";
+const OPEN_PARENTHESES = "(";
+const CLOSE_PARENTHESES = ")";
+const TRIPLE_DOT = "...";
+const COLON = ":";
+const EQUAL = "=";
+const AT = "@";
+const OPEN_SQUARE_BRACKET = "[";
+const CLOSE_SQUARE_BRACKET = "]";
```

Comment:
brackets are always 'square'. So we can simply call these OPEN_BRACKET and CLOSE_BRACKET